openseespy-pip/openseespy/postprocessing/internal_database_functions.py

```diff
@@ -293,3 +293,13 @@ def _readNodeDispData(ModelName,LoadCaseName):
     nodes_displacement = tempDisp
 	
     return timeSteps, nodes_displacement
+
+
+#### Read fibre data
+
+
+def recordFiberResponse(OutputName,eleNumber, sectionNumber):
```

Comment:
Where does it save the file? If this function is available to users, shouldn't it be in Get_Rendering() ?